### PR TITLE
Feature/rst 617 healthcheck notification

### PR DESCRIPTION
**Pull request by zaparka**

### CODE REVIEW

File: app/mailers/application_mailer.rb [hunk 1]
```diff
@@ -1,4 +1,12 @@
 class ApplicationMailer < ActionMailer::Base
   default from: Settings.mail.from
   layout 'mailer'
+
+  def dwp_is_down_notifier
+    to = Settings.mail.dwp_notification_alert
+    mail(to: to, subject: t('email.dwp_alert_notification..subject')) do |format|
+      format.html { render plain: t('email.dwp_alert_notification.message') }
```
Comment (garytaylor) [inline app/mailers/application_mailer.rb]: is this ok ? rendering plain text in html email ?

Comment (zaparka) [reply app/mailers/application_mailer.rb]: @garytaylor I was thinking about it but I couldn't find any reason why this should be an issue.

Comment (leoapost) [reply app/mailers/application_mailer.rb]: do we need the html format?

**Comment by leoapost** [comment]
Do we know what's the frequency of the notification when DWP is down?

**Comment by zaparka** [comment]
@leoapost we have to set up a cron job for this so it's up to us how often we call this.

File: app/mailers/application_mailer.rb (hunk 1 above)
Comment (zaparka) [reply app/mailers/application_mailer.rb]: @leoapost maybe we don't it theory it should be fine to render text only.

**Pull request merged**